R/hacker_news.r

```diff
@@ -1,24 +1,33 @@
 base_url <- "https://hacker-news.firebaseio.com/v0"
 
-#' get_top_stories
+get_item_ids_for_stories<-function(filename,limit=10){
+  fromJSON(paste(base_url, filename, sep=""))[1:limit]
+}
+
 #'
 #' This function returns the top stories from hacker news
 #'
 #' @param limit  The top limit stories to return
 #' @param pretty_print  If true return list of serialized stories else return the json structure
 #' @export
 #' @examples
-#' get_top_stories()
-#' get_top_stories(limit = 5, pretty_print = TRUE)
+#' get_stories()
+#' get_stories(limit = 10, pretty_print = TRUE)
```

Comment:
I think type is a required parameter.  In the build it runs these examples, so you may just need to add `type = "top"` here.